plugins/by-name/otter/default.nix

```diff
@@ -17,12 +17,8 @@ helpers.neovim-plugin.mkNeovimPlugin {
       [
         "plugins"
         "otter"
-        "addCmpSources"
       ]
-      "Otter is now supported by `plugins.cmp.autoEnableSources`, adding `otter` to `cmp` sources will enable this plugin."
     )
-    # Register nvim-cmp association
-    { cmpSourcePlugins.otter = "otter"; }
```

Comment:
I'm not sure the correct way to deprecate this. If a user is currently relying on `plugins.cmp.settings.sources` and `autoEnableSources`, then removing this association will mean otter is no longer enabled.

Reply:
IMHO, if a user has `plugins.cmp.settings.sources` and `autoEnableSources` in his config, he should get a warning.

Reply:
Yeah. I think this will require some manual handling in `plugins/cmp/auto-enable.nix`

Reply:
I've pushed up how I would suggest handling this. If you're happy you can squash this into your commit.

Reply:
> I've pushed up how I would suggest handling this. If you're happy you can squash this into your commit.

Yes, I am very happy with your suggestions! Thank you :-)